modules/ROOT/pages/troubleshoot-access-management.adoc

```diff
@@ -3,18 +3,18 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-These are some tips about what to do if you encounter these issues. 
+These are some tips about what to do if you encounter these issues.
 
 == Problems with Accessing Anypoint Platform
 
 === Sign in Credentials are not recognized
```

Comment:
Are these headings actual errors? If they are not, then use Title Case.

Reply:
```suggestion
=== Sign in Credentials are Not Recognized
```